upbound_uxp/manifest.json

```
@@ -0,0 +1,76 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f1427ca4-4b88-484a-a7c5-9cc66542b0d9",
+  "app_id": "upbound-uxp",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect and graph Upbound UXP metrics",
+    "title": "Upbound UXP",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Upbound UXP Overview Dashboard",
+        "image_url": "images/upbound_uxp_overview.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Upbound UXP Min Set Dashboard",
+        "image_url": "images/upbound_uxp_min_set.png"
+      }
+
+    ],
+    "classifier_tags": [
+      "Category::AWS",
+      "Category::Azure",
+      "Category::Cloud",
+      "Category::Configuration & Deployment",
+      "Category::Containers",
+      "Category::Developer Tools",
+      "Category::Google Cloud",
+      "Category::Kubernetes",
+      "Offering::Integration",
+      "Submitted Data Type::Metrics",
+      "Supported OS::Linux",
+      "Supported OS::macOS",
+      "Supported OS::Windows"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "Upbound UXP",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "uxp.",
+        "check": "uxp.can_connect",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "dashboards": {
+	"upbound_uxp_overview": "assets/dashboards/upbound_uxp_overview.json",
+	"upbound_uxp_min_set": "assets/dashboards/upbound_uxp_min_set.json"
+    },
+    "monitors": {
+	"[upbound] monitor controller runtime reconcoile error": ".assets/monitors/controller_runtime_reconcile_error.json"
```

Whoops, I missed this on my previous review but this path needs to be fixed
```suggestion
	"[upbound] monitor controller runtime reconcoile error": "assets/monitors/controller_runtime_reconcile_error.json"
```

---

Thanks for finding this. Updated.